weeter.gemspec

```diff
@@ -21,13 +21,13 @@ Gem::Specification.new do |s|
 
   s.add_dependency('eventmachine', '~> 1.2.0')
   s.add_dependency('eventmachine_httpserver', '~> 0.2.1')
-  s.add_dependency('em-hiredis', '~> 0.1.1')
-  s.add_dependency('multi_json', '~> 1.11.0')
+  s.add_dependency('em-hiredis', '~> 0.3.1')
+  s.add_dependency('multi_json', '~> 1.3.0')
```

Comment:
I had some trouble with that one. There were a lot of things in yapp-server that seems to want 1.3, but I don't know how it was working before, TBH.